linux_os/guide/services/ssh/ssh_server/sshd_use_approved_macs_ordered_stig/bash/shared.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_wrlinux,Red Hat Enterprise Linux 7,Oracle Linux 7
+
+if grep -q -P '^[[:space:]]*MACs[[:space:]]+' /etc/ssh/sshd_config; then
```

Comment:
FYI, you don't have to use `[[:space:]]` with `grep -P`, `\s` will do the same. And you could even copy-paste parts of the OVAL regex.